plugin-management-library/src/main/java/io/jenkins/tools/pluginmanager/util/PluginListParser.java

```diff
@@ -14,14 +19,15 @@
 import java.util.Arrays;
 import java.util.List;
 import org.apache.commons.lang3.StringUtils;
-import org.yaml.snakeyaml.Yaml;
```

Comment:
So we could remove snakeyamL?

Reply:
it's still used by jackson internally:
https://github.com/FasterXML/jackson-dataformats-text/tree/master/yaml

Reply:
removed the direct dep